### ISSUE TITLE
Transcript display should include in-line images or alt text

### ISSUE THREAD

**Issue by erkyrath** [erkyrath/lectrote#163]
Loading the image data might be tricky. It's in the original game file, not the transcript. In fact it might not be available at all. (The original game file might be gone, or we might have copied the .glktra file to a different machine entirely!)

The alt text should be available, though.

**Comment by erkyrath** [comment]
Implemented for next release (alt-text only).
